__tests__/bin.test.js

```diff
@@ -9,7 +9,7 @@ describe('bin', () => {
 
       await new Promise(done => {
         exec(`node ${__dirname}/../bin/rdme`, (error, stdout) => {
-          expect(stdout).toContain('a utlity for interacting with ReadMe');
+          expect(stdout).toContain('a utility for interacting with ReadMe');
```

Comment:
couldn't uodate the snapshot?

Reply:
problem is that the snapshot contains the package version, and we'd have to update the snapshot for every bump and I couldn't think of an easy way to get around that